docs/changes/coordinates/11444.feature.rst

```diff
@@ -0,0 +1,5 @@
+Add method ``transform`` to ``BaseRepresentation`` so that all subclasses can be
```

Comment:
I'd maybe start with "All representations now have a ``transform`` method, which allows them to be ... (the first part would seem an implementation detail)

Reply:
✅